src/features/profile/components/BeneficiaryHeader.tsx

```diff
@@ -15,30 +15,30 @@ type BeneficiaryHeaderProps = {
   domainsCredit?: DomainsCredit | null
   depositExpirationDate?: string
 }
+
 export function BeneficiaryHeader(props: PropsWithChildren<BeneficiaryHeaderProps>) {
+  const { firstName, lastName, domainsCredit, depositExpirationDate } = props
   return (
     <Container testID="beneficiary-header">
       <HeaderBackgroundWrapper>
         <HeaderBackground width={ScreenWidth} />
       </HeaderBackgroundWrapper>
       <Spacer.Column numberOfSpaces={12} />
       <UserNameAndCredit>
-        {/* eslint-disable-next-line react-native/no-raw-text */}
-        <Typo.Title4 color={ColorsEnum.WHITE}>{`${props.firstName} ${props.lastName}`}</Typo.Title4>
+        <Typo.Title4 color={ColorsEnum.WHITE}>{t`${firstName} ${lastName}`}</Typo.Title4>
```

Comment:
normalement tu devrais avoir des warning en dev, du type "Function are not valid as react child" avec ça.
Explication, t`message sans paramètre` => valide sans extraire les traductions.
Mais t`message avec {param}` n'est pas valide. Il faut d'abord changer la signature, et utiliser `t({id: 'id du messsage', values: {param}, message: 'message avec #param')` ET extraire les traductions

Reply:
pour te passer du warning, tu peux déclarer "const name = `${props.firstName} ${props.lastName}` " au dessus et simplement passer 'name'